java-lotto/src/main/java/domain/LottoGame.java

```diff
@@ -3,14 +3,17 @@
 
 import ui.OutputView;
 import ui.InputView;
+import ui.StringCalculator;
 
 
 public class LottoGame {
 
     public static void main(String[] args) {
 
         StringCalculator calculator = new StringCalculator(new InputView(), new OutputView());
-        String input = "1 + 1";
+        String input = "";
+        input = calculator.inputView.getInput();
```

Comment:
공백 선언없이 사용할 수 있겠네요 😄 
```suggestion
        String input = calculator.inputView.getInput();
```